frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -1,22 +1,39 @@
 import * as React from 'react';
+import { Split, SplitItem } from '@patternfly/react-core';
 import classNames from 'classnames';
+import { TFunction } from 'i18next';
 import { useTranslation } from 'react-i18next';
 import { SecondaryStatus } from '../../status';
 import { DashboardCardPopupLink } from '../dashboard-card/DashboardCardLink';
 import { HealthState, healthStateMapping, healthStateMessage } from './states';
 
+const renderAlerts = (alerts: string[], icon: React.ReactNode, t: TFunction): React.ReactNode =>
+  alerts.map((alert) => (
+    <Split hasGutter key={alert}>
+      <SplitItem>{icon}</SplitItem>
+      <SplitItem isFilled>{t(`ceph-storage-plugin~${alert}`)}</SplitItem>
```

Comment:
@afreen23 How do we "i18n" these dynamically generated strings? Should I manually add the string in the JSON? I can see some errors related to the generated string not being present in the JSON in the console.

Reply:
The texts from operator or backend are not covered for i18n.
Hence we do not need to externalize them.
```suggestion
      <SplitItem isFilled>{alert}</SplitItem>
```

Reply:
Thank you for clarifying this!